.github/workflows/docs.yml

```diff
@@ -9,7 +9,7 @@ on:
     types: [released]
   workflow_dispatch:
 env:
-  PYTHON_VERSION: "3.10"
+  PYTHON_VERSION: "3.10.7"
```

Comment:
We want to use the same Python version between the wheel building and consumption

Reply:
Is this a number that will need to be updated repeatedly? Manually tracked between GitHub Actions and ReadTheDocs?

Reply:
This number need only be synchronised within *this* workflow so that we build on the same interpreter as consumes the wheel. RTD doesn't consume this wheel, so it can do whatever it wants :)